frontend/public/components/namespace.jsx

```diff
@@ -968,14 +975,7 @@ export const NamespaceSummary = ({ ns }) => {
           <DetailsItem label={t('public~Status')} obj={ns} path="status.phase">
             <Status status={ns.status.phase} />
           </DetailsItem>
-          {canListSecrets && (
-            <>
-              <dt>{t('public~Default pull secret')}</dt>
-              <dd>
-                <PullSecret namespace={ns} />
-              </dd>
-            </>
-          )}
+          {canListSecrets && <PullSecret namespace={ns} />}
```

Comment:
With this change, we could show the secret values for users who can't list secrets since we read them from the service account. We'd just need to change it so that it's not a link.

Reply:
Changed and added screenshot above.